src/ContainerDeleteModal.jsx

```diff
@@ -17,7 +17,7 @@ const ContainerDeleteModal = (props) => {
                 </ModalBody>
                 <ModalFooter>
                     <Button color="secondary" onClick={props.handleCancelContainerDeleteModal}>Cancel</Button>
-                    <Button color="danger" onClick={props.handleRemoveContainer}>{_("Delete")}</Button>{' '}
+                    <Button id="container-delete-toggle-btn" color="danger" onClick={props.handleRemoveContainer}>{_("Delete")}</Button>{' '}
```

Comment:
I don't see this ID being used anywhere. Also, please avoid these. If you need to select these in tests, select the modal dialog first, and then the containing button. If it's just one, just `button` is enough, otherwise you have to be  more specific, like `button.btn-primary`.